app/code/Magento/Catalog/Test/Mftf/ActionGroup/GoToAttributeGridPageActionGroup.xml

```diff
@@ -10,7 +10,7 @@
               xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
     <actionGroup name="GoToAttributeGridPageActionGroup">
```

Comment:
There is a `deprecate` attribute available for Action Group.
Please use it (this is interpreted by the MFTF compiler and displayed at the top of results)

Reply:
@lbajsarowicz Done